Eniko/index.js

```diff
@@ -0,0 +1,168 @@
+       return "free shipping"
+   } else {
+       return "shipping costs apply"
+   }
+    
+};
+console.log(freeShipping(example4));
+console.log(freeShipping(example5));
+
+//--------------Or--------------
+// const freeShipping = obj => {
+// 	const valuesArray = Object.values(obj);
+// 	let sum = valuesArray.reduce((acc, curr) => acc + curr, 0);
+
+// OR a loop:
+// let sum = 0;
+// for (let i = 0; i < valuesArray.length; i++) {
+// 	sum += valuesArray[i];
+// }
+
+// 	return sum > 50 ? "Free shipping" : "Shipping costs apply";
+// }
+
+
+// freeShipping({ "Sponge": 3.50, "Soap": 5.99 }) ➞ false
+// freeShipping({ "Surround Sound Equipment": 499.99 }) ➞ true
+// freeShipping({ "Wool": 13.99, "Knitting Needles": 15.50, "Bag": 13.99 }) ➞ false
+// 6. Programming Object.
+
+const programming = {
+    languages: ["JavaScript", "Python", "Ruby"],
+    isChallenging: true,
+    isRewarding: true,
+    difficulty: 8,
+    jokes: "http://stackoverflow.com/questions/234075/what-is-your-best-programmer-joke"
+};
+// Write the command to add the language "Go" to the end of the languages array.
+programming.languages.push("go");
+console.log(programming);
+
+// Change the difficulty to the value of 7.
+programming.difficulty = 7;
+console.log(programming);
+// Using the delete keyword, write the command to remove the jokes key from the programming object.
+delete programming.jokes;
+console.log(programming);
+
+// Write a command to add a new key called isFun and a value of true to the programming object.
+programming.isfun = true;
+console.log(programming);
+
+// Using a loop, iterate through the languages array and console.log all of the languages.
+const languages = [];
+function getLanguages (arr) {
+    for (let i = 0; i < arr.length; i++) {
+        languages.push(arr[i]);
+    }
+    return languages;
+}
+
+console.log(getLanguages(programming.languages));
+// Using a loop, console.log all of the keys in the programming object.
+for (const key in programming) {
+    console.log(`keys ${key}`);
+}
+// Using a loop, console.log all of the values in the programming object.
+for (const key in programming) {
+    console.log("values", programming[key]);
+}
+// Create an object method where if the keys "isChallenging" and "isRewarding" have values of "true", then return "Learning the programming languages: "JavaScript, Python, Ruby, Go" is rewarding and challenging. Bonus: In a comment, explain what is printed if we console.log an object method without calling it and why.
+function isBothTrue () {
+    if (programming.isChallenging === true && programming.isRewarding === true) {
+        return `Learning the programming languages: "${programming.languages}" is rewarding and challanging.`
+    } else {
+            return `I give up.`
+    }
+};
+console.log(isBothTrue());
+
+// Bonus: It will be be shown as function, in that case not the function but the typeof the fucntion is called (my guess).
```

Comment:
Here is the solution of Bonus exercise. Using seal  object method we freeze the object, then if we try to delete
a key like for example languages, we cant!
Object.seal(programming);
delete programming.languages;
console.log("sealed ==>",programming)